### DISCUSSION TITLE
Specify a map extent for the map viewer

### DISCUSSION THREAD

**Discussion by emmanuelmathot** [Open-EO/openeo-web-editor#333] [Q&A]
Is there a way to specify the extent for the map viewer when clicking 'View on the map'?
I would expect that the extent in the `load_collection` node should be used if specified.

cc @vincentsarago

**Comment by m-mohr** [comment]
Unless the web service supports it or it's included in STAC metadata, there's no way to do that at this point. It would be good to have better integration though.
